React/Views/RCTView.m

```diff
@@ -99,6 +99,8 @@ - (UIView *)react_findClipView
 @implementation RCTView
 {
   UIColor *_backgroundColor;
+  NSMutableDictionary<NSString *, NSDictionary *> *accessibilityActionsNameMap;
```

Comment:
Not sure I understand why we need to have two maps here.

Reply:
@shergin In this design, actions have a non-localized name and a localized label.

in iOS, custom accessibility actions only have a localized label, which is what is passed to didActivateAccessibilityCustomAction. The label map is used to find the action name to include with the event to JS.

For standard actions like increment, decrement, and activate, we get specific calls from iOS (such as didActivate). We use the name map to validate whether the JS component claims to support that standard action.